unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatEntry.cs

```diff
@@ -89,26 +95,33 @@ public void Populate(Model chatEntryModel)
 
         Utils.ForceUpdateLayout(transform as RectTransform);
 
-        if (this.enabled)
+        if (processFading)
             group.alpha = 0;
     }
 
-    [SerializeField] CanvasGroup group;
+    public void OnPointerDown(PointerEventData pointerEventData)
+    {
+        if (model.messageType != ChatMessage.Type.PRIVATE) return;
+
+        OnPress?.Invoke(model.otherUserId);
+    }
 
     public void SetFadeout(bool enabled)
     {
         if (!enabled)
         {
             group.alpha = 1;
-            this.enabled = false;
+            processFading = false;
```

Comment:
maybe naming this as `fadeEnabled` is more clear? 

```suggestion
            fadeEnabled = false;
```

Reply:
Done